Update multi-level-menu.md (#1722)
pankod/refine: documentation/docs/guides-and-concepts/multi-level-menu/multi-level-menu.md

The `parentName` you give in the resource objects must be strictly equal to the resource name you want to group under.<br />
A resource given as a group can only have a `name` and a  `parentName`. They should not have other props such as list, edit, create etc.

> :::

Reviewer: ```suggestion
```

:::caution

> Because of the way next.js handles routing, you will need to use the `route` option to ensure routing continues to work when grouping resources under a parent.

Reviewer: ```suggestion
Since your Next.js applications are routing file-based, you need to manage the nested routes yourself. If you use Nested resources only for grouping Menu items in `Sider` and you don't need nested routes, you can give `route` option as a single level routing when defining your `resource`.
```

Reviewer: How do you think it looks like this?

:::
```tsx title="pages/_app.tsx"
        <Refine
           ...
            resources={[
                {
                    name: "CMS",
                },
                {
                    name: "posts",
                    parentName: "CMS",

>                     // highlight-start
>                     options: { route: "/posts" },
>                     // highlight-end

Reviewer: ```suggestion
                    // highlight-next-line
                    options: { route: "/posts" },
```

                    list: PostList,
                    create: PostCreate,
                    edit: PostEdit,
                    show: PostShow,
                },
                {
                    name: "category",
                    parentName: "CMS",

>                     // highlight-start
>                     options: { route: "/category" },
>                     // highlight-end

Reviewer: ```suggestion
                    // highlight-next-line
                    options: { route: "/category" },
```
